io/jvm/src/main/scala/fs2/io/JavaInputOutputStream.scala

```diff
@@ -191,7 +191,7 @@ private[io] object JavaInputOutputStream {
      * - DownStream signal -  keeps any remainders from last `read` and signals
      *                        that downstream has been terminated that in turn kills upstream
      */
-    Dispatcher[F].flatMap { dispatcher =>
+    Dispatcher.sequential[F].flatMap { dispatcher =>
```

Comment:
The `dispatcher` is used only for reading and closing. That seems sequential to me.